modules/remove.js

```diff
@@ -0,0 +1,2 @@
+const removeBook = (books, title) => books.filter((book) => book.title !== title);
```

Comment:
- [ ] You have done an excellent job creating the function to add and remove books :+1:. However, I have noticed that when a user tries to remove a book that has the same title as another book in the list, both books are removed. It is essential to fix this issue so that only the intended book is removed when the user clicks the remove button.


https://user-images.githubusercontent.com/105117832/222152915-5f505823-4d0b-4d7b-8538-9af4596c300c.mp4

- [OPTIONAL] It would be great if the form automatically reset to empty every time the user adds a book. This will provide a better user experience and save the user time from having to manually clear the form after each submission :+1: .